fiasco/elements.py

```diff
@@ -61,6 +61,12 @@ def element_name(self):
     def abundance(self):
         return self[0].abundance
 
+    @abundance.setter
+    def abundance(self, abundance):
+        for i in range(self.atomic_number + 1):
+            self[i]._abundance = abundance
+        self._abundance = abundance
+
```

Comment:
You do need the setter in `Element` if you want to type `iron.abundance = 1`, I think.  Otherwise, to change the abundance, you would need to call `iron[0].abundance = 1` since the setter would only be defined in `Ion`.  I think the former is a bit more intuitive.

Reply:
Ah, yes you're right. I for some reason missed the fact that this was the setter. I think you can still remove the last line though that sets `_abundance` since the abundance property already just pulls from the first ion. Also, I think you'll want to set `self[i].abundance` rather than `self[i]._abundance` so that the setter logic at the ion level still gets applied.